src/test/java/core/basesyntax/strategy/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static Storage fruitStorage;
+    private static OperationHandler handler;
+
+    @BeforeClass
+    public static void setup() {
+        fruitStorage = new Storage();

```

Comment:
we can put and get fruits directly from the map of the Storage class to avoid using Storage methods (which we don't need to test here) 
If it is not appliable for your implementation, let it be like now